src/world/message_handler.h

```diff
@@ -0,0 +1,18 @@
+#ifndef SERVER_MESSAGE_HANDLER_H
+#define SERVER_MESSAGE_HANDLER_H
+
+#include "common/mmo.h"
+#include <zmq.hpp>
+
+class MessageHandler
```

Comment:
I'd probably call this IMessageHandler, since its a pure interface

Reply:
Done